octobot_backtesting/api/exchange_data_collector.py

```diff
@@ -35,6 +35,8 @@ async def initialize_and_run_data_collector(data_collector):
     await data_collector.start()
     return data_collector.file_name
 
+async def stop_data_collector(data_collector):
+    return await data_collector.abort()
```

Comment:
`abort()` doesn't exist in `DataCollector` class, why not calling `datacollector.stop()` ?

Reply:
It was for my test, but right, it can be removed now